src/Joblify.Search/OfferSearchIndex.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Joblify.Search.Models;
+using Microsoft.Azure.Search;
+using Microsoft.Azure.Search.Models;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
```

Comment:
```suggestion
using System.Net;
```
and just use HttpStatusCode enum, without explicit namespace